vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/java/com/vaadin/flow/component/shared/ValidationUtil.java

```diff
@@ -170,4 +170,72 @@ public static <V extends Comparable<V>> ValidationResult validateMinConstraint(
         return isError ? ValidationResult.error(errorMessage)
                 : ValidationResult.ok();
     }
+
+    /**
+     * Checks if the value satisfies the minimum length constraint and returns a
+     * {@code ValidationResult.ok()} or {@code ValidationResult.error()} with
+     * the given error message depending on the result.
+     *
+     * @param errorMessage
+     *            the error message to return if the check fails
+     * @param value
+     *            the value to validate
+     * @param minLength
+     *            the minimum allowed length
+     * @return {@code ValidationResult.ok()} if the value is shorter than or
+     *         equal to the minimum length, {@code ValidationResult.error()}
+     *         otherwise
+     */
+    public static ValidationResult validateMinLengthConstraint(
+            String errorMessage, String value, Integer minLength) {
+        boolean isError = value != null && !value.isEmpty() && minLength != null
+                && value.length() < minLength;
+        return isError ? ValidationResult.error(errorMessage)
+                : ValidationResult.ok();
+    }
+
+    /**
+     * Checks if the value satisfies the maximum length constraint and returns a
+     * {@code ValidationResult.ok()} or {@code ValidationResult.error()} with
+     * the given error message depending on the result.
+     *
+     * @param errorMessage
+     *            the error message to return if the check fails
+     * @param value
+     *            the value to validate
+     * @param maxLength
+     *            the maximum allowed length
+     * @return {@code ValidationResult.ok()} if the value is longer than or
+     *         equal to the minimum length, {@code ValidationResult.error()}
```

Comment:
`minimum` -> `maximum`

Reply:
Thanks, fixed this and also another typo.